### ISSUE TITLE
SplitterLayout: screenreader does not always say the orientation

### ISSUE THREAD

**Issue by cmealares** [UI5/webcomponents-react#8232]
### Describe the bug

When a SplitterLayout is configured with attribute vertical=true, the screenreader (jaws) indicates the orientation.
When a SplitterLayout, is configured without specifying vertical (or with vertical=false), jaws does not indicate the orientation of the splitter.

I observe that in both cases, a correct aria-orientation property is generated in the DOM element.
My understanding of Jaws documentation is that it is supposed to give the orientation.
So, I suspect a problem in WC4R or UI5. 

### Isolated Example

WC4R's storybook

### Reproduction steps

1. Open WC4R storybook and go to the SplitterLayout example
2. Set focus on the splitter, jaws does not indicate the orientation.
3. Here is what is says:
```
storybook-preview-iframe frame
Press ARROW keys to move separator
Press ARROW keys to move min 0 max 100 current 50
```

1. Set SplitterLayout's vertical=true
2. Set focus on the splitter, jaws does indicate the orientation. 
3. Here is what is says:
```
storybook-preview-iframe frame
Press ARROW keys to move separator **Vertical**
Press ARROW keys to move min 0 max 100 current 50
```

### Expected Behaviour

Screenreader must always indicate the orientation.

### UI5 Web Components for React Version

V1 and V2

### UI5 Web Components Version

V1 and V2

### Browser

Chrome

### Operating System

Windows

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @cmealares 

As you already mentioned, `aria-orientation` is correctly set, so the announcement should be made by screen readers. 
I tested it with VoiceOver and there the current orientation is provided. I currently don't have access to JAWS, but when updating your version doesn't help, please file an issue in their repo.
